### DISCUSSION TITLE
Prisma Take not working

### DISCUSSION THREAD

**Discussion by cakctus** [prisma/prisma#17076] [Q&A]
I want to take first 10 record from the database
const data = await prisma.cars_model.findMany({
    take: 10,
    where: {
      brand_id: Number(brand),
      cars_generation: {
        some: {
          cars_modification: {
            some: {
              body_type: {
                equals: bodyType,
              },
            },
          },
        },
      },
    },
    include: {
      cars_generation: {
        include: {
          cars_modification: {
            where: {
              body_type: {
                equals: bodyType,
              },
            },
          },
        },
      },
    },
  }), 

and my result is 
![Screenshot_9](https://user-images.githubusercontent.com/39764706/210142522-d9c44f32-afa1-4a72-bf38-50fd25e3be0d.png)

Where is the problem ?

**Comment by nurul3101** [comment]
Hello @cakctus 👋 
Can you share your schema file so that I can try to run it locally and check if I get the same output?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
